### PR TITLE
working out displaying bugs

### PR DESCRIPTION
**Pull request by mel-ada**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -22,7 +21,6 @@ class App extends Component {
     return (
       <div className="App">
         <Layout cards={cards} />
-        <ReadingInfo />
       </div>
```
Comment (devonwesley) [inline src/App.js:51]: Just get rid of the extra white space

File: src/components/Card.js [hunk 2]
```diff
@@ -15,15 +15,19 @@ export default class Card extends Component {
     const  { card, faceUp } = this.props
```
Comment (devonwesley) [inline src/components/Card.js]: What is faceup? a boolean?

**Comment by devonwesley** [comment]
Does this PR fix the bug.??

File: src/components/Card.js (hunk 2 above)
Comment (mel-ada) [reply src/components/Card.js]: It's a boolean that helps determine if the back of the card or front of the card should be displayed

**Comment by mel-ada** [comment]
For clarification - the "bug" = displaying the back or front of the card at the right time. This doesn't fix it but we're getting closer.

**Pull request merged**